ui/app/src/Views/AlertList/Parsing.elm

```diff
@@ -4,19 +4,23 @@ import UrlParser exposing ((</>), (<?>), Parser, int, map, oneOf, parseHash, s,
 import Utils.Filter exposing (Filter, parseMatcher, MatchOperator(RegexMatch))
 
 
-boolParam : String -> UrlParser.QueryParser (Maybe Bool -> a) a
-boolParam name =
+parseSilenced : String -> UrlParser.QueryParser (Maybe Bool -> a) a
+parseSilenced name =
     UrlParser.customParam name
         (\x ->
             case x of
                 Nothing ->
-                    Nothing
```

Comment:
I think it is fine to have `Nothing` here to indicate that it was unset. Then we can later decide on what does it mean. Otherwise, it shouldn't be a `Maybe bool`.